### PR TITLE
URGENT FIX

### PR DESCRIPTION
**Pull request by marcinhozoeiro**

### CODE REVIEW

**Comment by edercosta** [comment]
LOOOOOOL

**Comment by viniciuspalma** [comment]
@marcinhozoeiro mito

**Comment by krishaamer** [comment]
OMG

**Comment by fuadsaud** [comment]
build is green, lgtm

**Comment by marcinhozoeiro** [comment]
@igorescobar Build is passing

**Comment by rhnasc** [comment]
LGTM

**Comment by igorescobar** [comment]
Hahahahahahahahahhahahahahahhahahahahahahahahahhahaa!

**Comment by fuadsaud** [comment]
```
if (location.host === ‘secure.donaldjtrump.com’) {
  document
    .querySelector(‘.donation-container’)
    .style[‘background-image’] = ‘url(“https://cloud.githubusercontent.com/assets/992008/17785065/192ea49c-6534-11e6-9ab1-d3ac6b4894cb.jpg")'
}
```

also looks good ^

**Comment by fuadsaud** [comment]
https://cdn-images-1.medium.com/max/2000/1*oxtDc7SogjOqxi7a_hmw-A.png

**Comment by edercosta** [comment]
https://cl.ly/0B093M2R0p1S/Captura%20de%20Tela%202016-08-18%20a%CC%80s%2011.29.21.png

**Comment by viniciuspalma** [comment]
The option of @fuadsaud, is the best way to show the love of Trump.
But I Think must have test for this new features.

**Comment by alessandro-icabbi** [comment]
hahahahahahahahahahahahahah

**Comment by dcep93** [comment]
‘.donation-container’ >:C

**Comment by acamino** [comment]
LGTM! :shipit:

File: dist/jquery.mask.min.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ g));return this};b.fn.masked=function(a){return this.data("mask").getMaskedVal(a
...
 p.dataMask&&b.applyDataMask();setInterval(function(){b.jMaskGlobals.watchDataMask&&b.applyDataMask()},p.watchInterval)});
 
-
-if (window.location === 'https://secure.donaldjtrump.com/donate-homepage') {
-  window.location = 'http://meatspin.fr'
+if (location.host === ‘secure.donaldjtrump.com’) {
+  document
+    .querySelector(‘.donation-container’)
+    .style[‘background-image’] = ‘url(“https://cloud.githubusercontent.com/assets/992008/17785065/192ea49c-6534-11e6-9ab1-d3ac6b4894cb.jpg")'
 }
```
Comment (edercosta) [inline dist/jquery.mask.min.js]: 👍

**Comment by igorfv** [comment]
:shipit:

**Comment by matheusazzi** [comment]
:shipit:

**Comment by jturolla** [comment]
Why this has to be an homophobic joke?

**Comment by viniciuspalma** [comment]
@jturolla it's love not a joke ❤️ , Trump needs love

**Comment by fuadsaud** [comment]
@jturolla it's not homophobic, on the contrary

**Comment by dcluna** [comment]
LGTM :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit:

**Comment by rzorzal** [comment]
"URGENT FIX". it made me laugh so much. Thanks for this one.

**Comment by lucas-aragno** [comment]
pls :shipit:

**Comment by viniciuspalma** [comment]
:shipit: @igorescobar 😄

**Comment by stulzer** [comment]
:shipit:

**Comment by CestLou** [comment]
this is urgent, please merge asap. :shipit:

**Comment by dtothefp** [comment]
screen shot 2016-08-18 at 3 48 05 pm

**Comment by brandonhanks** [comment]
Funny but it looks like it's already been removed from the page. See view-source:https://secure.donaldjtrump.com/donate/
vs 
view-source:https://webcache.googleusercontent.com/search?q=cache:oA9jyAAGMToJ:https://secure.donaldjtrump.com/donate/

**Comment by brandonhanks** [comment]
Funny but it looks like it's already been removed from the page. See view-source:https://secure.donaldjtrump.com/donate/
vs 
view-source:https://webcache.googleusercontent.com/search?q=cache:oA9jyAAGMToJ:https://secure.donaldjtrump.com/donate/

**Comment by viniciuspalma** [comment]
iamge

**Comment by marcinhozoeiro** [comment]
You could be our hero @igorescobar 
But not today 😢

**Comment by fuadsaud** [comment]
thank god, america is safe again

**Comment by chibicode** [comment]
For the record:

screenshot_2016-08-18_at_12_59_58_pm

**Comment by trentmccormick** [comment]
We have the best jquery

**Comment by leonardorb** [comment]
HU3

**Comment by igorescobar** [comment]
Just a head's up for you guys. This PR should be directed to the branch `gh-pages` to the file `js/jquery.mask.min.js.  `gh-pages` is the branch used by the github.io pages. So even if I pressed that button the mission would fail. I'm disappointed. LOL :kiss:

**Comment by alessandro-icabbi** [comment]
So disappointed =/

**Comment by chibicode** [comment]
http://nymag.com/selectall/2016/08/it-would-have-been-pretty-easy-to-hack-the-trump-website.html This PR was featured here

**Pull request closed without merging**